testing/zerotier-one/APKBUILD

```diff
@@ -28,9 +28,9 @@ package() {
 	cd "$builddir"
 	make DESTDIR="$pkgdir" install || return 1
```

Comment:
Please update second commit with removal all `return 1` here and in `build()`
Also commits should be squashed with message `testing/zerotier-one: upgrade to 1.2.10 and modernize`